src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/Specifications/TournamentsStorageSpecification.cs

```diff
@@ -22,11 +22,15 @@ public bool IsSatisfiedBy(TournamentEntity entity)
             var description = new ExpressionSpecification<TournamentEntity>(
                                 t => t.Description == null
                                   || t.Description.Length <= ValidationConstants.Tournament.MAX_DESCRIPTION_LENGTH);
+            var city = new ExpressionSpecification<TournamentEntity>(
```

Comment:
No need to update this in future. But you can leave this code here. It won't hurt